schema/filter_partners.js

```diff
@@ -13,7 +13,7 @@ import {
 const FilterPartners = {
   type: FilterPartnersType,
   description: 'Partners Elastic Search results',
-  args: _.create(Partners.args, {
+  args: _.extend(Partners.args, {
```

Comment:
either works for me, really

On Fri, Jan 22, 2016 at 6:11 PM, Laura Brown <notifications@github.com>
wrote:

> In schema/filter_partners.js
> <https://github.com/artsy/metaphysics/pull/97#discussion_r50603876>:
>
> > @@ -13,7 +13,7 @@ import {
> >  const FilterPartners = {
> >    type: FilterPartnersType,
> >    description: 'Partners Elastic Search results',
> > -  args: _.create(Partners.args, {
> > +  args: _.extend(Partners.args, {
>
> or
>
>   args: _.assign({
>     aggregations: {
>       type: new GraphQLNonNull(new GraphQLList(PartnersAggregation)),
>     },
>   }, Partners.args),
>
> less legible?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/artsy/metaphysics/pull/97/files#r50603876>.
>



-- 

[A.] Damon Zucconi

Artsy <http://artsy.net/>

damon@artsy.net

                917 974 9405 (Mobile)
                Twitter <http://twitter.com/dzucconi>